docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
-the native products with the Rackspace SIEM are included in the AWS
-infrastructure charges portion of the customer's Rackspace bill.
-
-Initial deployment and configuration will be charged as a one-time
-onboarding fee.
+Cloud Native Security service fees do not include the cost of the native security products themselves. The cost of the native products as well as the costs of the infrastructure supporting the integration with the Rackspace SIEM are included in the AWS infrastructure charges portion of the customer's Rackspace bill.
+
+Initial deployment and configuration will be charged as a one-time onboarding fee.
+
+Customers can choose between 2 billing models for Cloud Native Security – percentage of AWS infrastructure billing model or per-event billing model.
+
+Percentage of AWS infrastructure billing model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set percentage of their total AWS infrastructure costs on accounts that are in scope for Cloud Native Security. Charges will be based on usage of all AWS services and marketplace fees on accounts in scope for Cloud Native Security.
+
+In order to provide valuable service and to help with prioritization, a minimum severity level has been defined for each native AWS security product (source of findings). Rackspace will only ingest, and respond to, findings at or above the defined minimum severity level for each product. Findings with severities below this threshold will not be processed by Rackspace. The minimum severity levels for each product are defined as:
+
+* IAM Access Analyzer – **LOW**
+* Amazon GuardDuty – **MEDIUM**
+* AWS Security Hub (Security Standards) – **HIGH**
+
+Per-event billing model
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Customers that opt for this billing model will be charged a set fee for every security event Rackspace responds to. The monthly charge for the service is calculated as the product of the number of events multiplied by the cost per event, or the minimum monthly fee, whichever is greater.
+
+The per-event fee will apply to events (SIEM-correlated findings). For example, if the native security products generate five findings that are delivered to the SIEM and, after correlation, the SIEM generates two events (one for two related findings and one for three related findings), the customer will be charged for two events only.
+
+In order to give customers some control over costs (which have a direct correlation with the amount of findings Rackspace responds to) and to help with prioritization, we will define a minimum severity level for each customer. Rackspace will only respond to findings that are at or above the defined minimum severity level. Findings with lower severity level will not be processed by Rackspace and the customer won’t be charged for those. The minimum severity level will be agreed upon between the customer and Rackspace.
```

Comment:
... To give customers ...
... prioritization, we define a ...
... severity level. Rackspace doesn't process findings with a lower severity level and doesn't charge the customer for those.  The customer and Rackspace agree on the minimum security level. ...

Reply:
1 and 2 - updated
3 - to avoid having consecutive sentences that start with "Rackspace....", I replaced it with "Findings with lower severity level are not processed by Rackspace and the customer is not charged for those.".
Since the minimum severity level is something the customer and Rackspace **need** to agree on, I kept it unchanged.

Reply:
ok...i was trying to avoid the passive voice, which is why I put the actor first.